js/everything.js

```diff
@@ -529,11 +621,43 @@ $(function() {
 
     errorMessage.init();
 
+    videoMetadata = getParsedVideoMetadata();
+
+    if (videoMetadata) {
+        autoplayState = videoMetadata["autoplayState"];
+    }
+    if (autoplayState) {
+        updateAutoplayToggle(autoplayState);
+    }
+
     // How do we know if the value is truly invalid?
     // Preload the form from the URL
     var currentVideoID = getCurrentVideoID();
     if (currentVideoID) {
         $("#v").attr("value", currentVideoID);
+        // get similar videos, populate playList
+        if (!isFileProtocol()) {
+            $.ajax({
+                url: "https://www.googleapis.com/youtube/v3/search",
+                dataType: "json",
+                async: false,
+                data: {
+                    key: youTubeDataApiKey,
+                    part: "snippet",
+                    type: "video",
+                    relatedToVideoId: currentVideoID
+                },
+                success: function(data) {
+                    // push items into playlist
+                    for (var i = 0; i < data.items.length; i++) {
+                        playList.push(data.items[i].id.videoId);
+                    }
+                    playList = playList.reverse();
```

Comment:
Is there a reason to not iterate through `data.items` in reverse order? (ie: `for var i = data.items.length -; i >= 0; i--)`)

Reply:
Good point. I was using the code from the previous attempts, it's been changed now.